schemas/context/segmentdefinition-meta-base.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segment-meta-base",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segments basic metadata",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/segmentdefinition"],
+  "description": "Common fields required for any segment definition",
+  "definitions": {
+    "segment-meta-base": {
+      "properties": {
+        "xdm:entityIdentity": {
+          "title": "Segment Identity",
+          "$ref": "https://ns.adobe.com/xdm/context/segmentidentity",
+          "description": "Identity of the Segment"
+        },
+        "xdm:name": {
+          "title": "Name",
+          "type": "string",
+          "description": "Name of the Segment"
+        },
+        "xdm:description": {
+          "title": "Description",
+          "type": "string",
+          "description": "Description of the Segment"
+        },
+        "xdm:source": {
+          "title": "Source",
+          "type": "string",
+          "description": "Segment Owner. For example AAM, UPS etc"
+        },
+        "xdm:creationTime": {
+          "title": "Creation Time",
+          "type": "string",
+          "format": "date-time",
+          "description": ""
+        },
+       "xdm:updateTime": {
+          "title": "Update Time",
+          "type": "string",
+          "format": "date-time",
+          "description": ""
+        },
+        "xdm:status": {
+          "type": "string",
+          "title": "Status",
+          "description": "Current status of segment from external system normalized to ACTIVE/INACTIVE",
+          "enum" : [
```

Comment:
The `enum` field is generally used for validation. If you don't wish for the data to be validated explicitly this should move to a `meta:enum`

Reply:
Validation here is intended.